src/tic_tac_toe/ui.clj

```diff
@@ -0,0 +1,29 @@
+(ns tic-tac-toe.ui
+  (:require [tic-tac-toe.board :as board]))
+
+(def error-message "Bad input, please try again.")
+
+(defn init [fn-prompt fn-output]
+  (def prompt fn-prompt)
+  (def output fn-output))
```

Comment:
If somebody happens to not call this the rest of the functions would break, is this a problem?